package.json

```diff
@@ -52,8 +52,7 @@
     "express-mung": "^0.5.1",
     "isomorphic-fetch": "^2.2.1",
     "normalize-url": "^4.0.0",
-    "redis": "^2.8.0",
```

Comment:
please remove the packages via `yarn uninstall`, otherwise the package stays and `yarn.lock` and it isn't really removed